### ISSUE TITLE
Automate GeoJSON extraction of route-related data

### ISSUE THREAD

**Issue by roata** [roataway/web-ui#35]
We need a tool that automates the production of `route_X_stations.json` and `route_X_segments.json` files, such that they can be used by the web interface. At the moment we have data for 2 routes, and it was produced manually using a process described here: https://github.com/roataway/web-ui/blob/master/src/data/README.md

- This can be a Python script, placed in `tools/`
- It has to work from the command line
- It has to rely on data from https://github.com/roataway/web-ui/blob/master/src/data/routes.csv to extract IDs and generate file names
- The tool must be documented with a readme file

**Comment by iamandrewluca** [comment]
> This can be a Python script, placed in tools/
I recommend to use a node script

> It has to rely on data from /src/data/routes.csv@master  to extract IDs and generate file names
I recommend to move data from `src/data/routes.csv` to `JSON` format.  
Make a script that will generate a CSV file from that `JSON` when needed.  
CSV is more error prone than JSON, even current `src/data/routes.csv` has wrong syntax.

**Comment by roata** [comment]
1. I don't mind a script in node, but since this is just a standalone helper tool, I don't mind it being in another language either. A typical Linux distro will run a Python script out of the box, so that's a plus. (and a personal take on this - I can easily put it together in Python in a few minutes, whereas it would take me more time to figure out how to do this in JS).
2. The primary benefit of CSV is that it is much easier to handle by non-experts. If we want future data updates to come straight from RTEC, CSV will be easier for them because they can edit it in Excel while looking at a real table. Editing a JSON won't be as easy for an untrained person (who is specialized in transport logistics, but not in programming).

The syntax - I see that Github complains about the quotes, I will correct that when I update the route identifiers (I got all of them now).

**Comment by roata** [comment]
I think that an even easier approach for them would be to have a specific Google Sheets document that they can keep up to date. This way they get a 100% GUI-based solution and there is zero risk in accidentally forgetting to close a brace or escape a special symbol.

Are you aware of an easy way to accomplish that?

**Comment by roata** [comment]
This PR adds the data for the missing routes https://github.com/roataway/web-ui/pull/37/ 
I also included the tool I used to get the data, which can also deal with an individual route, without requiring a CSV file.

The route metadata probably changes once in a blue moon, so the only part where the presence of CSV is still relevant is `vehicles.csv`. If it ends up causing a lot of headaches, we'll address that in a separate ticket.
